lib/active_model/serializer/concerns/configuration.rb

```diff
@@ -10,6 +10,7 @@ module Configuration
         config = base.config
         config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
         config.serializer_lookup_enabled = true
+        config.on_serializer_not_found = -> {}
```

Comment:
See comments above about the effects (TLDR: none). When I set the default here, I thought that maybe someone will be looking at the code here to consider the values for `on_serializer_not_found`. I thought that an empty proc will communicate better that this was created to be (as opposed to `nil` which could be anything).